app/src/main/AndroidManifest.xml

```diff
@@ -22,13 +22,13 @@
         android:theme="@style/AppTheme">
 
         <activity
-            android:name=".ui.MainActivity"
+            android:name="com.xmartlabs.base.ui.MainActivity"
```

Comment:
But, the activities in the library should be generic (at least most of them) and therefore shouldn't be declared in the manifest